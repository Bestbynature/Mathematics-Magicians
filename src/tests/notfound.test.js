import { render } from '@testing-library/react';
import NotFound from '../components/notfound';
import '@testing-library/jest-dom/extend-expect';

describe('NotFound', () => {
  it('renders correctly', () => {
    const { container } = render(<NotFound />);
    expect(container).toMatchSnapshot();
  });

  it('renders the correct text', () => {
    const { getByText } = render(<NotFound />);
    expect(getByText('OOOOPS! Seems you entered a wrong URL!')).toBeInTheDocument();
  });
});
