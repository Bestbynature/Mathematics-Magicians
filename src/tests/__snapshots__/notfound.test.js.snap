// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound renders correctly 1`] = `
<div>
  <div
    class="not-found"
  >
    <h1
      data-testid="error-div"
    >
      OOOOPS! Seems you entered a wrong URL!
    </h1>
  </div>
</div>
`;
